pkgs/development/tools/mcap/mcap-cli/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "mcap-cli";
+  version = "0.0.28";
+
+  src = fetchFromGitHub {
+    owner = "foxglove";
+    repo = "mcap";
+    rev = "fd3ecada218f41929c6d430acba3561e5ebfaf08";
+    sha256 = "sha256-pIZThgb0wRmhNi3YrYTC+Tr6a/4Q7CWuHw+icjb1p8g=";
+  } + "/go/cli/mcap";
+
+  vendorHash = "sha256-3TSwOgTIaBg5U8UdQ74LmsS8gS05Yw02JjfaIN9fKQQ=";
+
+  # tests are currently failing because the test mcap files are in another directory in the repo
+  doCheck = false;
+
+  meta = with lib; {
+    description = "MCAP CLI";
```

Comment:
Could this be more descriptive? It doesn't really help somebody understand what this package does.